docs/changelog.md

```diff
@@ -138,6 +139,9 @@ permalink: /changelog/
   * Deprecated `PromptFeature` constructor that has parameters for both `Activity` and `Fragment`. Use the constructors that just take either one instead.
   * Changed `sessionId` parameter name to `customTabId`.
 
+* **sample-nearby-chat**
+  * 🆕 New sample program demonstrating use of `lib-nearby`.
+
```

Comment:
These entries should probably go together in the latest release version.

Reply:
Absolutely. I also fixed a typo on an earlier entry  when fixing mine.